### PR TITLE
Release/4.0.0-beta.7

### PR DESCRIPTION
**Pull request by revenuecat-ops**
- Seventh beta for RevenueCat framework 🎉
    100% Swift framework + ObjC support.
- See our [RevenueCat V4 API update doc](docs/V4_API_Updates.md) for API updates.
- macOS: improved ErrorCode.storeProblemError to indicate potential cancellation
    https://github.com/RevenueCat/purchases-ios/pull/943
- Log when duplicate subscription time lengths exist during Offering init
    https://github.com/RevenueCat/purchases-ios/pull/954
- PurchasesOrchestrator.paymentDiscount(forProductDiscount:product:completion:): improved error information
    https://github.com/RevenueCat/purchases-ios/pull/957
- Make a public originalData a thing for all our datatypes
    https://github.com/RevenueCat/purchases-ios/pull/956
- Detect ErrorCode.productAlreadyPurchasedError when SKError.unknown is actually cased by it
    https://github.com/RevenueCat/purchases-ios/pull/965

### CODE REVIEW

**Comment by NachoSoto** [comment]
🥳

File: CHANGELOG.latest.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
 - See our [RevenueCat V4 API update doc](docs/V4_API_Updates.md) for API updates.
-- Add more specific backend error subcodes
-    https://github.com/RevenueCat/purchases-ios/pull/927
+- macOS: improved ErrorCode.storeProblemError to indicate potential cancellation
+    https://github.com/RevenueCat/purchases-ios/pull/943
+- Log when duplicate subscription time lengths exist during Offering init
+    https://github.com/RevenueCat/purchases-ios/pull/954
+- PurchasesOrchestrator.paymentDiscount(forProductDiscount:product:completion:): improved error information
+    https://github.com/RevenueCat/purchases-ios/pull/957
+- Make a public originalData a thing for all our datatypes
```
Comment (vegaro) [inline CHANGELOG.latest.md:10]: it's rawData right?

File: CHANGELOG.latest.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+    https://github.com/RevenueCat/purchases-ios/pull/956
+- Detect ErrorCode.productAlreadyPurchasedError when SKError.unknown is actually cased by it
```
Comment (vegaro) [inline CHANGELOG.latest.md:12]: should this be "caused"

**Review by vegaro — CHANGES_REQUESTED** [review]

File: CHANGELOG.latest.md (hunk 2 above)
Comment (NachoSoto) [reply CHANGELOG.latest.md:12]: My bad 😇

Comment (taquitos) [reply CHANGELOG.latest.md]: Done! ✅

File: CHANGELOG.latest.md (hunk 1 above)
Comment (taquitos) [reply CHANGELOG.latest.md]: Done! ✅

**Review by vegaro — APPROVED** [review]

**Pull request merged**